Releasing.md

```diff
@@ -160,14 +156,6 @@ For the body of the post, just copy this checklist and again replace all occurre
 <p>o If there are any conflicts in the Gutenberg PR, merge <code>trunk</code> into it and resolve them.</p>
```

Comment:
I think we need a step like this for Gutenberg-Mobile somewhere too since there could be conflicts between the release branch and `trunk` there as well.

Reply:
Also, and this is not your change, but it looks like there are two `Merge the Gutenberg PR to <code>trunk</code>` steps (lines 160 and 168). I'm thinking the second one should be `Gutenberg-Mobile` instead of `Gutenberg`.